### PR TITLE
fix tutorial plot names

### LINKED ISSUE #211: confusion over plot.png vs confusion_matrix.png
https://github.com/iterative/example_cml/blob/master/train.py uses `plot.png` but the corresponding tutorial at https://cml.dev/doc/start/github uses `confusion_matrix.png`

Need to find all places this is out of sync (at least third time having this problem).

- [x] https://github.com/iterative/dvc.org/pull/3390
  - blog/2021-02-22-cml-runner-prerelease.md
  - blog/2020-07-16-devops-for-data-scientists.md
  - blog/2020-07-16-devops-for-data-scientists.md
- [ ] #212
  - public/page-data/doc/start/gitlab/page-data.json
  - public/page-data/doc/start/github/page-data.json
  - src/components/pages/Home/UseCasesSection/index.tsx
  - src/components/pages/Home/UseCasesSection/index.tsx
  - src/components/pages/Home/UseCasesSection/index.tsx
  - src/components/pages/Home/UseCasesSection/index.tsx
  - content/index.mdx
  - content/docs/start/github.md
  - content/docs/start/gitlab.md

### PR DESCRIPTION
**Pull request by casperdcl**
Fixes #211

### CODE REVIEW

**Review by 0x2b3bfa0 — APPROVED** [review]
Looks good to me

File: content/index.mdx [hunk 1]
```diff
@@ -185,8 +185,7 @@ cml:
     python train.py </Tooltip>
 
     <Tooltip type="reports"> - cat metrics.txt >> report.md - cml publish
-    confusion_matrix.png --md >> report.md - cml send-comment report.md
-    </Tooltip>
+    plot.png --md >> report.md - cml send-comment report.md </Tooltip>
```
Comment (casperdcl) [inline content/index.mdx:188]: a bit confused by the existing formatting here (and in the rest of this file) - @julieg18 are we missing some newlines?

Comment (julieg18) [reply content/index.mdx:188]: Yes, the formatting in this file is a little weird and hard to read, probably due to the mix of react components and markdown. Prettier helps in some sections but not everywhere. Sadly, I can't think of a way to improve it without manually formatting it. Is there an issue with the end result though? 

image

**Review by julieg18 — APPROVED** [review]
Looks good!

File: content/index.mdx (hunk 1 above)
Comment (casperdcl) [reply content/index.mdx:188]: #213 thanks

**Pull request merged**